frontends/web/src/components/keystoreconnectprompt.tsx

```diff
@@ -51,7 +52,7 @@ export function KeystoreConnectPrompt() {
           <SkipForTesting />
```

Comment:
would be good to add that also to the `wrongKeystore` error state, so one could try again unlocking also in devmode.

Reply:
Done